src/library/scala/util/Random.scala

```diff
@@ -43,11 +44,27 @@ class Random(val self: java.util.Random) extends AnyRef with Serializable {
    */
   def nextDouble(): Double = self.nextDouble()
 
+  /** Returns the next pseudorandom, uniformly distributed double value
+   *  between min (inclusive) and max (exclusive) from this random number generator's sequence.
+   */
+  def between(minInclusive: Double, maxExclusive: Double): Double = ThreadLocalRandom.current().nextDouble(minInclusive, maxExclusive)
```

Comment:
Hey @SethTisue the solution works for Float, Double and thankfully Int (since it has a `nextInt(n)`)

Using the formula `nextLong * (max - min) + min` would cause overflow if i'm not wrong !!
For `Int`, I've used `nextInt(max - min) + min`